events_task/lib/my_app_web/router.ex

```diff
@@ -0,0 +1,31 @@
+defmodule MyAppWeb.Router do
+  use MyAppWeb, :router
+
+  pipeline :browser do
+    plug(:accepts, ["html"])
+    plug(:fetch_session)
+    plug(:fetch_flash)
+    plug(:protect_from_forgery)
+    plug(:put_secure_browser_headers)
+
+    plug(Guardian.Plug.Pipeline,
+      module: MyAppWeb.Guardian,
+      error_handler: MyAppWeb.AuthController
+    )
+
+    plug(Guardian.Plug.VerifySession)
+    plug(Guardian.Plug.LoadResource, allow_blank: true)
+  end
+
+  scope "/", MyAppWeb do
+    pipe_through(:browser)
+    resources("/events", EventController)
+    get("/confirm", ConfirmationController, :create)
```

Comment:
Get requests should be used only for idempotent requests (in general - requests that are not changing state of server/DB).
It's because they can be cached, and most of the tools treat them as requests that can be safely repeated.
So here confirming should be post or put.